R/tmb.R

```diff
@@ -107,15 +122,25 @@ h_mmrm_tmb_formula_parts <- function(formula) {
     model_formula,
     stats::as.formula(paste(". ~ . +", subject_var, "+", visit_var))
   )
+  if (!is.null(group_var)) {
+    full_formula <- stats::update(
+      full_formula,
+      stats::as.formula(sprintf(". ~ . + %s", group_var))
+    )
+  }
+  ret <- list(
+    formula = formula,
+    model_formula = model_formula,
+    full_formula = full_formula,
+    cov_type = deparse(cov_term[[1L]]),
+    visit_var = visit_var,
+    subject_var = subject_var
+  )
+  if (!is.null(group_var)) {
+    ret <- c(ret, list(group_var = group_var))
+  }
```

Comment:
```suggestion
    subject_var = subject_var,
    group_var = group_var
  )
```
easier to just always include it - downstream when we check for `NULL` this is still correct